### PR TITLE
ULS Get Started: add password manager support

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref: #404 
WP PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/14831

This adds support for password managers. 
- 1Password from the email field.
  - I have disabled the iOS version check to facilitate testing.
- Password auto-fill from the keyboard.
  - To make this work, I implemented the same workaround as `LoginEmailViewController`. That is, I added a "hidden" text field in the view to catch the password from autofill. When the password field changes, login is attempted.

**Note to self** - before merging, re-enable `setupOnePasswordButtonIfNeeded` version check.

managers

### CODE REVIEW

**Review by mindgraffiti — APPROVED** [review]

File: WordPressAuthenticator/Unified Auth/View Related/Reusable Views/TextFieldTableViewCell.swift [hunk 1]
```diff
@@ -126,16 +128,16 @@ private extension TextFieldTableViewCell {
     /// Sets up a 1Password button if 1Password is available and user is on iOS 12.
     ///
     @objc func setupOnePasswordButtonIfNeeded() {
-        if #available(iOS 13, *) {
-            // no-op, we rely on the key icon in the keyboard to initiate a password manager.
-        } else {
+//        if #available(iOS 13, *) {
+//            // no-op, we rely on the key icon in the keyboard to initiate a password manager.
+//        } else {
```
Comment (Gio2018) [inline WordPressAuthenticator/Unified Auth/View Related/Reusable Views/TextFieldTableViewCell.swift]: can we remove the commented out lines here and below?

**Review by Gio2018 — CHANGES_REQUESTED** [review]
Hey @ScoutHarris , this looks good! I just noticed a small chunk of commented-out-code but that's it.

**Comment by ScoutHarris** [comment]
Hey @Gio2018 .

That was intentional, as noted in the description:
> I have disabled the iOS version check to facilitate testing.
> Note to self - before merging, re-enable setupOnePasswordButtonIfNeeded version check.

Sorry, I forgot to add a TODO. But I _just_ re-enabled it, so it worked out. 😄

**Review by Gio2018 — APPROVED** [review]
:shipit:!

**Pull request merged**